app/controllers/concerns/classified_listings_toolkit.rb

```diff
@@ -28,11 +28,6 @@ def bump_listing
   end
 
   def clear_listings_cache
-    cb = CacheBuster.new
-    cb.bust("/listings")
-    cb.bust("/listings?i=i")
-    cb.bust("/listings/#{@classified_listing.category}/#{@classified_listing.slug}")
-    cb.bust("/listings/#{@classified_listing.category}/#{@classified_listing.slug}?i=i")
-    cb.bust("/listings/#{@classified_listing.category}")
+    ClassifiedListings::BustCacheJob.perform_later(@classified_listing.id)
```

Comment:
The cache used to be busted synchronously, but not it's busted async. I'm not sure if it makes a difference, but for safety, I would keep the old behavior by using `perform_now`

Reply:
Or just `CacheBuster.bust_classified_listings`